src/otg/datasource/finngen/finngen_finemapping.py

```diff
@@ -0,0 +1,234 @@
+                f.col("lbf_variable4").cast("double").alias("lbf_4"),
+                f.col("lbf_variable5").cast("double").alias("lbf_5"),
+                f.col("lbf_variable6").cast("double").alias("lbf_6"),
+                f.col("lbf_variable7").cast("double").alias("lbf_7"),
+                f.col("lbf_variable8").cast("double").alias("lbf_8"),
+                f.col("lbf_variable9").cast("double").alias("lbf_9"),
+                f.col("lbf_variable10").cast("double").alias("lbf_10"),
+            )
+            .withColumn(
+                "posteriorProbability",
+                f.when(f.col("credibleSetIndex") == 1, f.col("alpha_1"))
+                .when(f.col("credibleSetIndex") == 2, f.col("alpha_2"))
+                .when(f.col("credibleSetIndex") == 3, f.col("alpha_3"))
+                .when(f.col("credibleSetIndex") == 4, f.col("alpha_4"))
+                .when(f.col("credibleSetIndex") == 5, f.col("alpha_5"))
+                .when(f.col("credibleSetIndex") == 6, f.col("alpha_6"))
+                .when(f.col("credibleSetIndex") == 7, f.col("alpha_7"))
+                .when(f.col("credibleSetIndex") == 8, f.col("alpha_8"))
+                .when(f.col("credibleSetIndex") == 9, f.col("alpha_9"))
+                .when(f.col("credibleSetIndex") == 10, f.col("alpha_10")),
+            )
+            .drop(
+                "alpha_1",
+                "alpha_2",
+                "alpha_3",
+                "alpha_4",
+                "alpha_5",
+                "alpha_6",
+                "alpha_7",
+                "alpha_8",
+                "alpha_9",
+                "alpha_10",
+            )
+            .withColumn(
+                "logBF",
+                f.when(f.col("credibleSetIndex") == 1, f.col("lbf_1"))
+                .when(f.col("credibleSetIndex") == 2, f.col("lbf_2"))
+                .when(f.col("credibleSetIndex") == 3, f.col("lbf_3"))
+                .when(f.col("credibleSetIndex") == 4, f.col("lbf_4"))
+                .when(f.col("credibleSetIndex") == 5, f.col("lbf_5"))
+                .when(f.col("credibleSetIndex") == 6, f.col("lbf_6"))
+                .when(f.col("credibleSetIndex") == 7, f.col("lbf_7"))
+                .when(f.col("credibleSetIndex") == 8, f.col("lbf_8"))
+                .when(f.col("credibleSetIndex") == 9, f.col("lbf_9"))
+                .when(f.col("credibleSetIndex") == 10, f.col("lbf_10")),
+            )
+            .drop(
+                "lbf_1",
+                "lbf_2",
+                "lbf_3",
+                "lbf_4",
+                "lbf_5",
+                "lbf_6",
+                "lbf_7",
+                "lbf_8",
+                "lbf_9",
+                "lbf_10",
+            )
+        )
+
+        # drop credible sets where logbf < 2. Except when there's only one credible set in region:
+        # 0.8685889638065036 corresponds to np.log10(np.exp(2)), to match the orginal threshold in publication.
+        finngen_finemapping_summaries_df = (
```

Comment:
I kind of don't understand what this file contains and why you have to join it with SuSIEs outputs. Can you make that clearer in the parameter docstrings?